### PR TITLE
updated bibtex info

### PR DESCRIPTION
**Pull request by danielru**
Updates bibtex entry for DDM2013 paper; removed ISDC paper as it is not PINT: moved to SDC references bibfile.

### CODE REVIEW

**Comment by danielru** [comment]
I won't add any more commits, I promise :) So this PR is ready to be merged.

File: _bibliography/pint.bib [hunk 1]
```diff
@@ -2525,3 +2511,70 @@ @phdthesis{Beck2016
...
+        editor="Dickopf, Thomas and Gander, J. Martin and Halpern, Laurence and Krause, Rolf and Pavarino, F. Luca",
+        chapter="Parareal for Diffusion Problems with Space- and Time-Dependent Coefficients",
+        title="Domain Decomposition Methods in Science and Engineering {XXII}",
+        year="2016",
+        publisher="Springer International Publishing",
+        pages="371--378",
+        doi="10.1007/978-3-319-18827-0_37",
+        url="http://dx.doi.org/10.1007/978-3-319-18827-0_37"
+}
+
```
Comment (pancetta) [inline _bibliography/pint.bib]: Hm, this doesn't look right in the reference list..

File: _bibliography/pint.bib [hunk 2]
```diff
@@ -2525,3 +2511,70 @@ @phdthesis{Beck2016
...
+@Inbook{RuprechtEtAl2016,
+	abstract = {{For the time-parallel Parareal method, there exists both numerical and analytical proof that it converges very well
+for diffusive problems like the heat equation. Many applications, however, do not lead to simple homogeneous
+diffusive problems but feature strongly inhomogeneous and possibly anisotropic coefficients. In the talk, we will
+present results from a numerical study of how non-constant coefficients in a diffusion problem influence the convergence
+behaviour of Parareal. Further, the effect of different parameters like e.g. temporal and spatial resolution
+and geometry is explored.}},
+        author="Ruprecht, Daniel and Speck, Robert and Krause, Rolf",
```
Comment (torbjoernk) [inline _bibliography/pint.bib]: don't use quotation marks but curly brackets (also below)

File: _bibliography/pint.bib [hunk 3]
```diff
@@ -2518,6 +2492,18 @@ @article{Wu2015b
 % 2016
 %
 
+@article{AlhubailEtAl2016,
+        title = {The swept rule for breaking the latency barrier in time advancing {PDEs}},
```
Comment (torbjoernk) [inline _bibliography/pint.bib]: do you mind sorting the fields alphabetically?

File: _bibliography/pint.bib [hunk 4]
```diff
@@ -2525,3 +2511,70 @@ @phdthesis{Beck2016
...
+	author = {Bolten, Matthias and Moser, Dieter and Speck, Robert},
+	title = {A multigrid perspective on the parallel full approximation scheme in space and time},
+	url = {http://arxiv.org/abs/1603.03586},
+	howpublished = {arXiv:1603.03586 [math.NA]},
+	year = {2016},
+	abstract = {{For the numerical solution of time-dependent partial differential equations, time-parallel methods have recently shown to provide a promising way to extend prevailing strong-scaling limits of numerical codes. One of the most complex methods in this field is the "Parallel Full Approximation Scheme in Space and Time" (PFASST). PFASST already shows promising results for many use cases and many more is work in progress. However, a solid and reliable mathematical foundation is still missing. We show that under certain assumptions the PFASST algorithm can be conveniently and rigorously described as a multigrid-in-time method. Following this equivalence, first steps towards a comprehensive analysis of PFASST using block-wise local Fourier analysis are taken. The theoretical results are applied to examples of diffusive and advective type.}}
+}
+
+@book{FalgoutEtAl2016, 
+	place={United States},
```
Comment (torbjoernk) [inline _bibliography/pint.bib]: I'm nitpicking, I know. space around equal sign maybe?

**Comment by pancetta** [comment]
See, these comments are the reason why people do not want to contribute.. they check the last few commits and then say like "nah..". Just kidding..

**Comment by torbjoernk** [comment]
Ok, I shut up :no_mouth:

**Comment by pancetta** [comment]
@danielru Are you going to edit this?

**Comment by danielru** [comment]
Yes, it is on my list :)

**Comment by danielru** [comment]
Okay, I hope I fixed everything - if so, merge, if not, complain ;)

**Pull request merged**